web/README.md

```diff
@@ -0,0 +1,64 @@
+## Getting started
+
+- Download [Hugo](https://gohugo.io/#action) and put it in your `PATH`
+- Install Node packages with `npm install` (or via `yarn`), *OPTIONAL* for SCSS and Javascript
```

Comment:
Below there is a yarn.lock file. If yarn is optional, how is it updated? --> for discussion.